Python/ceval.c

```diff
@@ -763,8 +763,6 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
 
     DISPATCH();
 
-handle_eval_breaker:
-
     /* Do periodic things, like check for signals and async I/0.
```

Comment:
Maye this comment is now out of date? It appears that the call to `_Py_HandlePending()` is now inside `CHECK_EVAL_BREAKER()`.